.travis.yml

```diff
@@ -37,6 +37,11 @@ matrix:
       script:
         - ./travis/onnx_integration_tests.sh
 
+    - name: "Pytorch Artifact"
+      python: "3.6"
+      script:
+        - ./travis/run-keras-tests.sh
```

Comment:
do you mean to name it `run-pytorch-tests.sh`?

Reply:
yeah sry about that
in the issues page the script was named as such